src/titiler/core/titiler/core/errors.py

```diff
@@ -57,7 +54,6 @@ def exception_handler_factory(status_code: int) -> Callable:
     """
 
     def handler(request: Request, exc: Exception):
-        logger.error(exc, exc_info=True)
```

Comment:
IMO: error message will be forwarded to the client so we don't really need to `print` it

Reply:
@geospatial-jeff, I've decided to completely remove the logger because ☝️. If this is wrong I'll make the change you are proposing!